### PR TITLE
Feature/gaps adjustments

### PR DESCRIPTION
**Pull request by Trjegul84**

### CODE REVIEW

File: src/pages/search/drawer/species/richness/NumberOfSpecies.jsx [hunk 1]
```diff
@@ -339,14 +339,18 @@ class NumberOfSpecies extends React.Component {
...
-                  <div>
-                    <a href="http://biomodelos.humboldt.org.co" target="_blank" rel="noopener noreferrer">
-                      <Icon image={biomodeloslink} hoverImage={biomodeloslink2} />
-                    </a>
-                    {/* TODO:
-                    Add I2D link when it's ready (import mappointlink and mappointlink2 images)
-                    */}
-                  </div>
+                  {(filter === 'inferred') && (
+                  <>
```
Comment (erikasv) [inline src/pages/search/drawer/species/richness/NumberOfSpecies.jsx:343]: This is not necessary, put only the div

File: src/pages/search/drawer/species/richness/NumberOfSpecies.jsx [hunk 2]
```diff
@@ -339,14 +339,18 @@ class NumberOfSpecies extends React.Component {
...
+                    <div>
+                      <a href="http://biomodelos.humboldt.org.co" target="_blank" rel="noopener noreferrer">
+                        <Icon image={biomodeloslink} hoverImage={biomodeloslink2} />
+                      </a>
+                      {/* TODO:
+                       Add I2D link when it's ready (import mappointlink and mappointlink2 images)
```
Comment (erikasv) [inline src/pages/search/drawer/species/richness/NumberOfSpecies.jsx:349]: this indentation is wrong

File: src/pages/search/drawer/species/richness/NumberOfSpecies.jsx [hunk 3]
```diff
@@ -339,14 +339,18 @@ class NumberOfSpecies extends React.Component {
...
+                       */}
+                    </div>
+                  </>
+                    )}
```
Comment (erikasv) [inline src/pages/search/drawer/species/richness/NumberOfSpecies.jsx:353]: this indentation is also wrong

File: src/pages/search/drawer/species/richness/SpeciesRecordsGaps.jsx [hunk 4]
```diff
@@ -181,6 +188,12 @@ class SpeciesRecordsGaps extends React.Component {
           />
           )
         )}
+        {showErrorMessage && (
+          <div className="disclaimer">
+            Los vacíos en el área de consulta son mayores o menores que los de la región biótica
```
Comment (erikasv) [inline src/pages/search/drawer/species/richness/SpeciesRecordsGaps.jsx:193]: This applies for min and max values, maybe we should express that in the message

**Review by erikasv — CHANGES_REQUESTED** [review]

File: src/pages/search/drawer/species/richness/SpeciesRecordsGaps.jsx (hunk 4 above)
Comment (Trjegul84) [reply src/pages/search/drawer/species/richness/SpeciesRecordsGaps.jsx:193]: New message is just as Jaime said.

File: src/pages/search/drawer/species/richness/NumberOfSpecies.jsx (hunk 3 above)
Comment (Trjegul84) [reply src/pages/search/drawer/species/richness/NumberOfSpecies.jsx]: deleted

**Review by erikasv — APPROVED** [review]

**Pull request merged**